### PR TITLE
Creation of Key Value placeholder

### PR DESCRIPTION
**Pull request by FlaviooLima**
| - | - |
| --- | --- |
| Issue | No issue, but necessary and part of  https://github.com/ripe-tech/ripe-robin-revamp/issues/72 |
| Dependencies | - |
| Decisions | Creation of a placeholder for Key Value component.<br> To animate this component is required the Placeholder context that can be found in https://github.com/ripe-tech/ripe-robin-revamp/pull/141 <br> This context will provide constant update of the gradient locations. |
| Animated GIF | Apr-13-2020 12-10-51 |

### CODE REVIEW

File: react/components/molecules/key-value-placeholder/key-value-placeholder.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            <View style={styles.sectionViewPlaceholder}>
+                <View style={styles.textContainer}>
+                    <LinearGradient
+                        style={styles.key}
+                        angle={90}
+                        useAngle={true}
+                        colors={this.props.colors}
+                        locations={this.props.locations}
+                    />
+                    {this._renderLines()}
```
Comment (gcandal) [inline react/components/molecules/key-value-placeholder/key-value-placeholder.js:52]: Take a look at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill for a slightly shorter way to do this.

File: react/components/molecules/key-value-placeholder/key-value-placeholder.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        paddingHorizontal: 16,
+        borderBottomWidth: 1,
+        borderBottomColor: "#e4e8f0"
+    },
+    textContainer: {
+        flex: 1
+    },
+    key: {
+        borderRadius: 4,
+        height: 18.5,
```
Comment (gcandal) [inline react/components/molecules/key-value-placeholder/key-value-placeholder.js:72]: Let's go with `18`, if we can't spot the difference let's stay with integers.

File: react/components/molecules/key-value-placeholder/key-value-placeholder.stories.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import { storiesOf } from "@storybook/react-native";
+import { withKnobs, number } from "@storybook/addon-knobs";
+
+import { KeyValuePlaceholder } from "./key-value-placeholder";
+
+storiesOf("Atoms", module)
+    .addDecorator(withKnobs)
+    .add("Key Value Placeholder", () => {
+        const numberOfLines = number("Number of lines", 1);
+        const placeholderColors = ["#DBDEE5", "#E3E6ED", "#DBDEE5"];
```
Comment (joamag) [inline react/components/molecules/key-value-placeholder/key-value-placeholder.stories.js:11]: Use lowercased hexadecimal please

**Comment by joamag** [comment]
@gcandal please don't forget to post your review result

File: react/components/molecules/key-value-placeholder/key-value-placeholder.stories.js (hunk 3 above)
Comment (FlaviooLima) [reply react/components/molecules/key-value-placeholder/key-value-placeholder.stories.js:11]: Vscode was claiming that lowercase was wrong, I'm gonna fix this and my configs.

File: react/components/molecules/key-value-placeholder/key-value-placeholder.js (hunk 1 above)
Comment (FlaviooLima) [reply react/components/molecules/key-value-placeholder/key-value-placeholder.js:52]: Has per our conversation this suggestion has no effect now.

**Pull request merged**